pkg/repo/index.go

```diff
@@ -140,20 +140,32 @@ func (i Index) Get(name, version string) (string, error) {
 // If the name is not found, this will return false.
 func (i Index) GetVersions(name string) ([]BundleVersion, bool) {
 	ret, ok := i[name]
-	rawversions := []string{}
-	for ver := range ret {
-		rawversions = append(rawversions, ver)
+	if !ok {
+	Loop:
```

Comment:
nit: do you have strong feelings about keeping the label here?
As it is not required in `for` statements, and not widely used, it might confuse people not familiar with labels.

But this is not necessarily something keeping this PR from getting in.

Reply:
I'm not immediately seeing a simple way to break through the doubly-nested loop here. My understanding of reading the [Golang spec for `break` statements](https://golang.org/ref/spec#Break_statements) is that it only drops out of the inner-most loop if not specified with a loop label. That is why I have included the `Loop` label on the outer `for` loop.

Reply:
I guess you could extract the nested loop into a separate function and directly return when you found it.